houston/constants.go

```diff
@@ -0,0 +1,9 @@
+package houston
+
+// Houston Constants
+const (
+	// Deployment Level Role Based Access
+	DEPLOYMENT_ADMIN  = "DEPLOYMENT_ADMIN"
```

Comment:
DEPLOYMENT_ADMIN -> `DeploymentAdmin = "DEPLOYMENT_ADMIN" to respect go style guide.